archive-product.php

```diff
@@ -11,33 +11,31 @@
     <div class="row margin-bottom-small">
 <?php
 if( have_posts() ) {
-  $i = 0;
   while( have_posts() ) {
     the_post();
 
     $product = new WC_Product($post->ID);
     $in_stock = $product->is_in_stock();
     $availability = $product->get_availability();
 
-    if ($i % 4 === 0 && $i !== 0) {
-      echo '</div>\n<div class="row margin-bottom-small">';
-    }
 ?>
-    <article <?php post_class('col col3 shop-product'); ?> id="product-<?php the_ID(); ?>">
+    <article <?php post_class('col col-s col-s-12 col-m col-m-6 col-l col-l-3 shop-product'); ?> id="product-<?php the_ID(); ?>">
       <a href="<?php the_permalink() ?>">
         <div class="shop-product-title-holder">
           <h3 class="shop-product-title font-serif font-italic">
             <?php the_title(); ?>
           </h3>
         </div>
         <?php if (!$in_stock) { ?>
-          <span class="shop-product-sold font-uppercase"><?php echo $availability['availability']; ?></span>
+          <span class="row align-center shop-product-sold font-uppercase">
+            <img src="<?php echo get_stylesheet_directory_uri() . '/img/dist/teardrop.svg'; ?>">
```

Comment:
you can include the SVG inline via `file_get_contents` to save a request. we dont have a threshold for usage vs size on this but i imagine better to inline but up to you